src/modules/rlm_rest/rlm_rest.c

```diff
@@ -1096,6 +1096,31 @@ static int parse_sub_section(rlm_rest_t *inst, CONF_SECTION *parent, CONF_PARSER
 	return 0;
 }
 
+static int _mod_conn_free(fr_curl_io_request_t *randle)
+{
+	curl_easy_cleanup(randle->candle);
+	return 0;
+}
+
+static int rest_conn_alloc(fr_curl_io_request_t *randle, void *uctx)
+{
+	rlm_rest_t const	*inst = talloc_get_type_abort(uctx, rlm_rest_t);
+	rlm_rest_curl_context_t	*curl_ctx = NULL;
+
+	randle->candle = curl_easy_init();
+	if (!randle->candle) return -1;
```

Comment:
Either MEM() or wrap the condition in unlikely() and say what the error was with `fr_strerror`